README.md

```diff
@@ -94,6 +97,29 @@ An additional dimension that is not `spatial`, but may be `temporal` if the data
 
 An Additional Dimension Object MUST specify an `extent` or a set of `values`. It MAY specify both.
 
+### Variable Object
+
+A *Variable Object* defines a variable (or a multi-dimensional array). The variable may have dimensions, which are described by [Dimension Objects](#dimension-object).
+
+| Field Name       | Type                         | Description |
+| ---------------- | -----------------------------| ----------- |
+| dimensions       | \[string]                    | **REQUIRED.** The dimensions of the variable. This should refer to keys in the ``cube:dimensions`` object or be an empty list if the variable has no dimensions. |
+| type             | string                       | **REQUIRED.** Type of the variable, either `data` or `auxiliary`. | 
+| description      | string                       | Detailed multi-line description to explain the dimension. [CommonMark 0.29](http://commonmark.org/) syntax MAY be used for rich text representation. |
+| extent           | \[number\|string\|null]      | If the dimension consists of [ordinal](https://en.wikipedia.org/wiki/Level_of_measurement#Ordinal_scale) values, the extent (lower and upper bounds) of the values as two-dimensional array. Use `null` for open intervals. |
```

Comment:
Hmm, a string extent would be usually for date-times, right? On the other hand, a step can't be specified as duration like in the Temporal Dimension. The definition of the Variable Object seems it tries to merge all kinds of types for which we have different Dimension Objects into a single Object, but that doesn't feel very clean. Maybe we need to split this up a bit more or so?

Reply:
Yes, if JSON / json schema models dates as strings then having strings here seems necessary. So I think `step` should be allowed to be a string (to be more precise, it should be an ISO 8601 duration if and only if extent is a datetime string, but I'm not sure how to write that in JSON schema).

That said, I don't think that `step` is too useful for variables (they're extremely useful for dimensions / coordinates). 

I'll defer to you on the best way to model this, but having a single JSON object for all variables feels natural to me. When you're using the data they all end up in n-dimensional arrays, just with different data types.

Reply:
So maybe we can simplify here and remove the step for now from variables? I still need to think through whether to split the Variables Object or not...